euth_wagtail/assets/scss/components/_dashboard.scss

```diff
@@ -139,6 +139,14 @@ h1.dashboard-title {
     background-color: #fff;
     text-align: center;
     padding: 1em;
+
+    > button {
```

Comment:
According to our [style guide ](http://codeguide.co/#css-selectors) styling tags directly is bad style. Also this is the main reason why, you are using a descendant selector (`<`) here. In this particular example, this could probably also be resolved by using the bootstrap provided `btn` classes.